api/trakt-auth/api/src/main/java/app/tivi/trakt/TraktManager.kt

```diff
@@ -36,7 +35,6 @@ import me.tatarka.inject.annotations.Inject
 class TraktManager(
     private val dispatchers: AppCoroutineDispatchers,
     private val showTasks: ShowTasks,
-    private val traktClient: Lazy<TraktV2>,
     private val authStore: AuthStore,
 ) {
```

Comment:
We aren't storing the bearer token in the client anymore. Furthermore, it should be delivered from AuthStore directly.